### ISSUE TITLE
クリッカブルな部品に使用する色と装飾等の取り決め

### ISSUE THREAD

**Issue by knokmki612** [npocccties/chiloportal#507]
Figma https://www.figma.com/file/dCE06JShf29eqnvZ4vcE8U?node-id=731:6679#294315206

![image](https://user-images.githubusercontent.com/9744580/199938630-00acf5d3-01b8-4764-bab5-02e4aadc7f74.png)

> デザインシステムとして記載しておくと良さそうな内容についてですが、
> クリッカブルな部品に使用する色と装飾、またはクリッカブルな部品に使用してはならない色と装飾が定義されていると操作の一貫性に関わる部分なのでより良さそうに思いました。
> 
> ---
> 
> Jumpu UI に基礎的なルールが必要だったりしますね
> 
> ---
> 
> 特定のプロジェクトに限定しない形でのJumpu UIでのルール決めは困難なように思います。限定しないで言えるのは、たとえばリンクならばリンクであることを期待できる色にしましょうなどかと思います。
> プロジェクトごとにプライマリカラーが押せそうな色としてつかえるかどうかはその色に依存するかと思います。
> ただ、Jumpu UIの設計としてはプライマリカラーはおせそうな色の想定になっていますから、まあそれに従うということではあるのでしょうか。
> 気になっているのはグレー系の色でリンクを表現する箇所があるので、それについては何かしら明記しておいたほうがいいかと思った次第です。
> もっとも、今後（現状のトップ画面）ではそのような色づかいは廃止されるということであれば特段明記は不要かもしれません。

**Comment by ties-makimura** [comment]
#614 より

  - [ ] リンク付き文字に黒色と赤色が混在している
    案：本文とパンクズのリンクについては全て赤色に統一する．フッターのリンクについては黒色に統一する
  - [ ] リンク付き文字に，アンカーがあるものとないものが混在している
    案：全てアンカーなしにして，マウスオーバー時にアンカーを表示する．

**Comment by ties-makimura** [comment]
ボタン風なデザインも，場所によってクリックできたりできなかったりで分かりにくいです．
特に，能力バッジページにある所は，マウスカーソルまで変わるのにクリックしても反応しないのでバグっているように見えました．
![image](https://user-images.githubusercontent.com/8957521/203723939-cfbce807-032d-46fd-9c83-aee753c601a0.png)

**Comment by knokmki612** [comment]
> 能力バッジページにある所は，マウスカーソルまで変わるのにクリックしても反応しないのでバグっているように見えました．

はい。 `jumpu-tag` というスタイルがデフォルトではクリッカブルであることを想定しているために生じています。一部のコンポーネントではクリッカブルな見た目を無効化していますが、それ以外の部分でも同様の対応が必要な状態かと思います。

https://github.com/npocccties/chiloportal/search?q=jumpu-tag&type=code

**Comment by ties-makimura** [comment]
> 一部のコンポーネントではクリッカブルな見た目を無効化していますが、それ以外の部分でも同様の対応が必要な状態かと思います。

それ以外の部分については，https://github.com/npocccties/chiloportal/issues/614#issuecomment-1326052863 の検討事項と重なるので，そちらも考慮して取り決めれればと思います．

**Comment by Hidetaro7** [comment]
> 本文とパンクズのリンクについては全て赤色に統一する．フッターのリンクについては黒色に統一する

パンくずについて
あまり色を付けすぎると重要なコンテンツとの区別をわかりにくくさせてしまうだけでなく、派手な印象を与えてしまったりすることも考えられるので、パンくずについては現状の色とさせてください。
機能として充分果たしていると考えられます。

本文を赤色
良いと思います。現状そうなっていると見受けられます。

> * 案：全てアンカーなしにして，マウスオーバー時にアンカーを表示する．

アンカーとは下線のことを指していますか？
赤色の本文中リンクはデフォルトで下線なし、hover時に下線付きでよいかと思います。

@ties-makimura

**Comment by ties-makimura** [comment]
> パンくずについて あまり色を付けすぎると重要なコンテンツとの区別をわかりにくくさせてしまうだけでなく、派手な印象を与えてしまったりすることも考えられるので、パンくずについては現状の色とさせてください。 機能として充分果たしていると考えられます。

了解です．

> アンカーとは下線のことを指していますか？ 赤色の本文中リンクはデフォルトで下線なし、hover時に下線付きでよいかと思います。

それでお願いします．

**Comment by knokmki612** [comment]
https://github.com/npocccties/oku-private/issues/171 にてリンクに関するガイドラインが示されました。実際にはリンクの見た目は他にもいくつかバリアントが存在しますが、それについてはFigmaを確認してください。取り決めについては存在し明文化された状態ですのでクローズします。本件に関して疑問あればFigmaにてコメントをつけるなどしてもらえればと思います。
